requirements/nlp.txt

```diff
@@ -22,5 +22,5 @@ stanza
 subword_nmt>=0.3.8
 termcolor
 tokenizers
-transformers>=4.12.0
+transformers>=4.12.0,<=4.30.2
```

Comment:
why limit the version of transformers?

Reply:
It's a mistake in merging 2 different git branches and had been fixed.